mlibrary/lib.umich.edu
# monthly package updates

# Overview
Monthly dependency update to help maintain lib.umich.edu.

## NPM
These dependencies have been updated to their latest versions:
- `gatsby`
- `gatsby-plugin-image`
- `gatsby-plugin-manifest`
- `gatsby-plugin-sharp`
- `gatsby-plugin-sitemap`
- `gatsby-source-drupal`
- `gatsby-source-filesystem`
- `gatsby-transformer-sharp`
- **`use-debounce`** (major version)
- **`whatwg-url`** (major version)
- `eslint`
- `eslint-plugin-react`
- `gatsby-plugin-netlif`

[https://future-wwwlib-previews.netlify.app/](https://future-wwwlib-previews.netlify.app/)

## Testing
- Install the updated packages (`npm install`).
- Start [the site](http://localhost:8000) and check if everything still works (`npm start`).
- Make sure the PR is consistent in these browsers:
